src/main/kotlin/org/rust/lang/utils/RustEscapesUtils.kt

```diff
@@ -32,7 +32,7 @@ fun parseRustStringCharacters(chars: String, outChars: StringBuilder, sourceOffs
 
     val outOffset = outChars.length
     var index = 0
-    chars.tokenize(RustEscapesLexer(STRING_LITERAL, true, true)).forEach {
+    chars.tokenize(RustEscapesLexer(STRING_LITERAL, true, true, true)).forEach {
```

Comment:
Ah, now I get it :smile: That `STRING_LITERAL` in `parseRustStringCharacters` is only there because I had to pass something as an argument. We don't care what default token type is used in this method, we only need offsets of the escapes. This is exactly the same situation as in `unescapeRust`.

I'll put some comments with explanation for future generations.